main.tf

```diff
@@ -15,47 +19,64 @@ resource "aws_rds_cluster" "this" {
   vpc_security_group_ids          = concat([aws_security_group.this.id], var.additional_security_groups)
   tags                            = var.tags
   db_cluster_parameter_group_name = var.db_cluster_parameter_group_name
-  deletion_protection             = true
-}
-
-resource "aws_secretsmanager_secret" "root_password" {
-  name_prefix = "aurora-root-${var.name}"
-  description = "Root password for the ${var.name} aurora cluster database"
-  tags        = var.tags
-}
-
-resource "aws_secretsmanager_secret_version" "root_password" {
-  secret_id     = aws_secretsmanager_secret.root_password.id
-  secret_string = random_password.password.result
-}
+  deletion_protection             = var.deletion_protection
+  copy_tags_to_snapshot           = true
 
-resource "random_password" "password" {
-  length           = 32
-  special          = true
-  min_special      = 1
-  override_special = "-._~" # URL-safe characters prevent parsing errors when using this password in a connection string
+  enabled_cloudwatch_logs_exports = [
+    "postgresql",
+  ]
```

Comment:
We probably want to look into a sane default for a parameter group so that we can log slow queries from inception.